basic-sql-operations.md

```diff
@@ -1,174 +1,156 @@
 ---
-title: Explore SQL with TiDB
+title: Basic SQL operations
 summary: Learn about the basic SQL statements for the TiDB database.
 aliases: ['/docs/dev/basic-sql-operations/','/docs/dev/how-to/get-started/explore-sql/']
 ---
 
-# Explore SQL with TiDB
+# Basic SQL Operations
 
 TiDB is compatible with MySQL, you can use MySQL statements directly in most of the cases. For unsupported features, see [Compatibility with MySQL](/mysql-compatibility.md#unsupported-features).
 
-To experiment with SQL and test out TiDB compatibility with MySQL queries, you can [run TiDB directly in your web browser without installing it](https://tour.tidb.io/). You can also first deploy a TiDB cluster and then run SQL statements in it.
+Structured Query Language (SQL) is a declarative language which defines the way that users interact with databases. You can use SQL just as a natural language to talk to the database. This page walks you through the basic SQL statements in TiDB. For a complete list of TiDB statements, see [TiDB SQL Syntax Diagram](https://pingcap.github.io/sqlgram/).
 
-This page walks you through the basic TiDB SQL statements such as DDL, DML and CRUD operations. For a complete list of TiDB statements, see [TiDB SQL Syntax Diagram](https://pingcap.github.io/sqlgram/).
+## Category
 
-## Create, show, and drop a database
+SQL is divided into the following 4 types according to their functions:
 
-### Create a database
+- DDL (Data Definition Language): It is used to define database objects, including databases, tables, views, and indexes.
 
-To create a database, use the `CREATE DATABASE` statement:
+- DML (Data Manipulation Language): It is used to manipulate application related records.
 
-{{< copyable "sql" >}}
+- DQL (Data Query Language)：It is used to query the records after conditional filtering.
 
-```sql
-CREATE DATABASE db_name [options];
-```
+- DCL (Data Control Language): It is used to define access privileges and security levels
 
-For example, to create a database named `samp_db`:
+Common DDL features are creating, modifying, and deleting objects (such as tables and indexes). The corresponding commands are `CREATE`, `ALTER`, and `DROP`.
 
-{{< copyable "sql" >}}
+## Show, create and drop a database
 
-```sql
-CREATE DATABASE IF NOT EXISTS samp_db;
-```
-
-### Show the databases
+Database in TiDB can be considered as a collection of objects such as tables and indexes.
 
-To show the databases, use the `SHOW DATABASES` statement:
+To show the list of databases, use the `SHOW DATABASES` statement:
 
 {{< copyable "sql" >}}
 
 ```sql
 SHOW DATABASES;
 ```
 
-### Delete a database
-
-To delete a database, use the `DROP DATABASE` statement:
+Use the database named `mysql`:
```

Comment:
Please keep the sentence structure consistent with that of others.
Use > To use